src/main/java/seedu/duke/logic/commands/favcommand/AddFavCommand.java

```diff
@@ -43,6 +44,7 @@ private void addToFavList() {
         if (!FavList.contains(item)) {
             FavList.addFav(item);
             addFavMessage(input);
+            isValid = true;
```

Comment:
Should we allow users to add any fav commands to the favorite list? I have assigned isValid to false for the /descfav command

Reply:
> Should we allow users to add any fav commands to the favorite list? I have assigned isValid to false for the /descfav command

I asked ethan about this ytd, he say for all fav commands isvalid=false

Reply:
I think if we are going to prevent users from adding "redundant commands" that would include all fav commands.

Reply:
i should remove addfav from being valid hahaha